CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Add `hidden` functionality to Enroll page. The Admin can enable the functionality by changing the value of `Hide Enrollment Flow` field to true, in the config page of an Enrollment flow. By default the value is false/empty and all the configured Enrolment Flows will be displayed in `People->Enroll` page.
 - Retrieve AuthenticatingAuthority and depict in CO Person's canvas/profile
 - IdP hinting for RCAUTH plugin
+- Redirect User to the SP after registration. Currently the User was stack in COmanage profile view and should go back and reselect the service.
```

Comment:
the user was redirected to their COmanage profile

Reply:
Done